src/app/layout/layout.component.css

```diff
@@ -1,7 +1,56 @@
-.layout-wrapper{
+::ng-deep :root {
+  --link-on-navbar-text-hover-color: #00bcd4;
+  --link-on-navbar-to-sign-up-color: #CFD8DC;
+  --link-on-navbar-to-sign-up-hover-color: #B0BEC5;
+}
+
+.layout-wrapper {
   height: 100%;
 }
 
-.flex-wrapper{
+.flex-wrapper {
   height: 100%;
 }
+
+mat-toolbar {
+  position: fixed;
+  top: 0;
+}
+
+.img {
+  width: auto;
+  height: auto;
+  float: left;
+}
+
+.fill-remaining-space {
+  flex: 1 1 auto;
+}
+
+a {
+  display: inline-block;
+  padding: 0.3125rem 1.25rem;
+  margin: 0 0.625rem;
+  color: white;
+  font-size: 1.125rem;
+  text-decoration: none;
+}
+
+mat-toolbar a:hover {
+  color: var(--link-on-navbar-text-hover-color);
+}
+
+#link--to-sign-up {
+  background: var(--link-on-navbar-to-sign-up-color);
```

Comment:
link-on-navbar-to-sign-up-color-might-be-better-named,-like--navbar-link-sign-up-color
Because it has a hierarchy like this navbar > link > sign-up. DO NOT USE on and to. Also naming should reflect logical element hierarchy